C:37:"PHPUnit\Runner\DefaultTestResultCache":7155:{a:2:{s:7:"defects";a:26:{s:7:"Warning";i:6;s:43:"AT\Exceptable\Tests\HandlerTest::testDuring";i:2;s:42:"AT\Exceptable\Tests\HandlerTest::testError";i:2;s:46:"AT\Exceptable\Tests\HandlerTest::testException";i:2;s:45:"AT\Exceptable\Tests\HandlerTest::testShutdown";i:2;s:42:"AT\Exceptable\Tests\HandlerTest::testThrow";i:2;s:40:"AT\Exceptable\Tests\HandlerTest::testTry";i:3;s:39:"AT\Exceptable\Tests\HandlerTest::testOn";i:4;s:66:"AT\Exceptable\Tests\IsExceptableTest::testGetInfo with data set #0";i:3;s:66:"AT\Exceptable\Tests\IsExceptableTest::testGetInfo with data set #1";i:3;s:69:"AT\Exceptable\Tests\IsExceptableTest::testGetBadInfo with data set #0";i:3;s:65:"AT\Exceptable\Tests\IsExceptableTest::testCreate with data set #0";i:4;s:65:"AT\Exceptable\Tests\IsExceptableTest::testCreate with data set #1";i:4;s:98:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with code only"";i:4;s:96:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with context"";i:4;s:119:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with context and previous exception"";i:3;s:107:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with previous exception"";i:4;s:99:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with code only"";i:4;s:97:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with context"";i:4;s:120:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with context and previous exception"";i:3;s:108:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with previous exception"";i:4;s:66:"AT\Exceptable\Tests\IsExceptableTest::testHasInfo with data set #0";i:4;s:66:"AT\Exceptable\Tests\IsExceptableTest::testHasInfo with data set #1";i:4;s:69:"AT\Exceptable\Tests\IsExceptableTest::testNotHasInfo with data set #0";i:4;s:54:"AT\Exceptable\Tests\IsExceptableTest::testLocalization";i:4;s:78:"AT\Exceptable\Tests\IsExceptableTest::testLocalization with data set "stubbed"";i:2;}s:5:"times";a:51:{s:7:"Warning";d:0;s:43:"AT\Exceptable\Tests\HandlerTest::testDuring";d:0;s:42:"AT\Exceptable\Tests\HandlerTest::testError";d:0;s:46:"AT\Exceptable\Tests\HandlerTest::testException";d:0;s:45:"AT\Exceptable\Tests\HandlerTest::testShutdown";d:0;s:42:"AT\Exceptable\Tests\HandlerTest::testThrow";d:0;s:40:"AT\Exceptable\Tests\HandlerTest::testTry";d:0.003;s:39:"AT\Exceptable\Tests\HandlerTest::testOn";d:0;s:66:"AT\Exceptable\Tests\IsExceptableTest::testGetInfo with data set #0";d:0;s:66:"AT\Exceptable\Tests\IsExceptableTest::testGetInfo with data set #1";d:0;s:69:"AT\Exceptable\Tests\IsExceptableTest::testGetBadInfo with data set #0";d:0;s:65:"AT\Exceptable\Tests\IsExceptableTest::testCreate with data set #0";d:0;s:65:"AT\Exceptable\Tests\IsExceptableTest::testCreate with data set #1";d:0;s:98:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with code only"";d:0.001;s:96:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with context"";d:0;s:119:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with context and previous exception"";d:0;s:107:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "UNKNOWN_FOO with previous exception"";d:0;s:99:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with code only"";d:0;s:97:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with context"";d:0;s:120:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with context and previous exception"";d:0;s:108:"AT\Exceptable\Tests\IsExceptableTest::testNewExceptable with data set "TOO_MUCH_FOO with previous exception"";d:0;s:101:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "UNKNOWN_FOO with code only"";d:0;s:99:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "UNKNOWN_FOO with context"";d:0;s:122:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "UNKNOWN_FOO with context and previous exception"";d:0;s:110:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "UNKNOWN_FOO with previous exception"";d:0;s:102:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "TOO_MUCH_FOO with code only"";d:0;s:100:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "TOO_MUCH_FOO with context"";d:0;s:123:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "TOO_MUCH_FOO with context and previous exception"";d:0;s:111:"AT\Exceptable\Tests\IsExceptableTest::testCreateExceptable with data set "TOO_MUCH_FOO with previous exception"";d:0;s:100:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "UNKNOWN_FOO with code only"";d:0;s:98:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "UNKNOWN_FOO with context"";d:0;s:121:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "UNKNOWN_FOO with context and previous exception"";d:0;s:109:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "UNKNOWN_FOO with previous exception"";d:0;s:101:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "TOO_MUCH_FOO with code only"";d:0;s:99:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "TOO_MUCH_FOO with context"";d:0;s:122:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "TOO_MUCH_FOO with context and previous exception"";d:0;s:110:"AT\Exceptable\Tests\IsExceptableTest::testThrowExceptable with data set "TOO_MUCH_FOO with previous exception"";d:0;s:66:"AT\Exceptable\Tests\IsExceptableTest::testHasInfo with data set #0";d:0;s:66:"AT\Exceptable\Tests\IsExceptableTest::testHasInfo with data set #1";d:0;s:69:"AT\Exceptable\Tests\IsExceptableTest::testNotHasInfo with data set #0";d:0;s:95:"AT\Exceptable\Tests\IsExceptableTest::testGetInfo with data set "__TestExceptable::UNKNOWN_FOO"";d:0;s:96:"AT\Exceptable\Tests\IsExceptableTest::testGetInfo with data set "__TestExceptable::TOO_MUCH_FOO"";d:0;s:95:"AT\Exceptable\Tests\IsExceptableTest::testHasInfo with data set "__TestExceptable::UNKNOWN_FOO"";d:0;s:96:"AT\Exceptable\Tests\IsExceptableTest::testHasInfo with data set "__TestExceptable::TOO_MUCH_FOO"";d:0;s:80:"AT\Exceptable\Tests\IsExceptableTest::testIs with data set "same class and code"";d:0;s:87:"AT\Exceptable\Tests\IsExceptableTest::testIs with data set "same class, different code"";d:0;s:80:"AT\Exceptable\Tests\IsExceptableTest::testIs with data set "subclass, same code"";d:0;s:86:"AT\Exceptable\Tests\IsExceptableTest::testIs with data set "non-exceptable, same code"";d:0;s:91:"AT\Exceptable\Tests\IsExceptableTest::testIs with data set "non-exceptable, different code"";d:0;s:54:"AT\Exceptable\Tests\IsExceptableTest::testLocalization";d:0;s:78:"AT\Exceptable\Tests\IsExceptableTest::testLocalization with data set "stubbed"";d:0;}}}